WcaOnRails/config/locales/carrierwave.sv.yml

```diff
@@ -0,0 +1,31 @@
+sv:
+  errors:
+    messages:
+      #original_hash: e0f0ec5
```

Comment:
FTR. we don't expect the `carrierwave` and `time_will_tell` files to change and there's no particular need for the Internationalize metadata, but it's also fine to keep it :)